eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelData.scala

```diff
@@ -466,22 +466,24 @@ case class LocalParams(nodeId: PublicKey,
                        fundingKeyPath: DeterministicWallet.KeyPath,
                        dustLimit: Satoshi,
                        maxHtlcValueInFlightMsat: UInt64, // this is not MilliSatoshi because it can exceed the total amount of MilliSatoshi
-                       channelReserve: Satoshi,
+                       requestedChannelReserve_opt: Option[Satoshi],
                        htlcMinimum: MilliSatoshi,
                        toSelfDelay: CltvExpiryDelta,
                        maxAcceptedHtlcs: Int,
                        isInitiator: Boolean,
                        defaultFinalScriptPubKey: ByteVector,
                        walletStaticPaymentBasepoint: Option[PublicKey],
-                       initFeatures: Features[InitFeature])
+                       initFeatures: Features[InitFeature]) {
+  val requestedChannelReserve: Satoshi = requestedChannelReserve_opt.getOrElse(0 sat)
```

Comment:
This is fishy, I expected the field `requestedChannelReserve_opt` to be made `private` (or with a comment to discourage its direct use) and the following method in `Commitments`:

```
val requestedChannelReserve: Satoshi = requestedChannelReserve_opt.getOrElse(1 % * capacity)
```

Reply:
> But I can remove the requestedChannelReserve field entirely, and let callers use commitments.xxxChannelReserve or when we have no commitments do the getOrElse themselves, what do you think?

Yes that works, we always have a `Commitments`, except in `Helpers.validate*`, where it makes sense to be more manual anyway.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2237/commits/47cb396bd8216c843038cc8e80d2244b7afa5424
I will need to make sure the dual funding PRs correctly compute the reserve in `makeFirstCommitTxs`.